Firestore/core/src/firebase/firestore/remote/exponential_backoff.cc

```diff
@@ -32,6 +32,17 @@ using firebase::firestore::util::AsyncQueue;
 using firebase::firestore::util::TimerId;
 namespace chr = std::chrono;
 
+/**
+ * Initial backoff time in milliseconds after an error. Set to 1s according to
+ * https://cloud.google.com/apis/design/errors.
+ */
+constexpr Milliseconds kDefaultBackoffInitialDelay = Milliseconds(1000);
```

Comment:
Is the second `Milliseconds` required? Could this be `constexpr Milliseconds kDefaultBackoffInitialDelay = 1000;`?

Reply:
I don't think so -- they've been declared with the second `Milliseconds` elsewhere in the code base. Removing it results in the following error:
`No viable conversion from 'int' to 'const firebase::firestore::remote::Milliseconds' (aka 'const duration<long long, ratio<1LL, 1000LL> >')`

Reply:
Gotcha. Yep keep it around then :-).